db/fixtures/miq_product_features.yml

```diff
@@ -1837,6 +1837,15 @@
       :description: View Policy Profiles
       :feature_type: view
       :identifier: miq_policy_set_view
+      :children:
+      - :name: List
+        :description: Display All Policy Profiles
+        :feature_type: view
+        :identifier: miq_policy_set_show_list
```

Comment:
Maybe we would need to change it [here](https://github.com/ManageIQ/manageiq-api/blob/master/config/api.yml#L2485) too and add a migration.